### ISSUE TITLE
Lost of image align

### ISSUE THREAD

**Issue by emics** [jomjol/AI-on-the-edge-device#1646]
### The Problem

hi i've install the version 13.0.7 and after update to 13.0.8
after a full configuration the system work fine but after some hour report error during image analisys. I open the Overview page and the image is turned, not completly aligned.

![image](https://user-images.githubusercontent.com/1004363/208646262-1cb59b40-1c11-462f-bc42-10103a4a66c5.png)

if I reboot the image come back to correct orientation and the reading analisys is Perfect.

in the attached log the Round 20 stil OK - in the Round 21 the image is rotated.

### Version

13.0.8

### Logfile

```shell
[0d01h35m20s] 2022-12-20T11:19:03 <INF> [TFLITE SERVER] Round #20 started
[0d01h35m20s] 2022-12-20T11:19:03 <DBG> [MQTT IF] Published topic: watermeter/status, content: Take Image (msg_id=292)
[0d01h35m27s] 2022-12-20T11:19:10 <DBG> [MQTT IF] Published topic: watermeter/status, content: Aligning (msg_id=293)
[0d01h35m41s] 2022-12-20T11:19:24 <DBG> [MQTT IF] Published topic: watermeter/status, content: Digitalization of ROIs (msg_id=294)
[0d01h35m41s] 2022-12-20T11:19:24 <DBG> [CNN] doFlow nach Alignment
[0d01h35m42s] 2022-12-20T11:19:25 <DBG> [TFLITE] Make Allocate
[0d01h35m42s] 2022-12-20T11:19:25 <DBG> [CNN] Processing Number 'main'
[0d01h35m42s] 2022-12-20T11:19:25 <DBG> [CNN] ROI #0 - TfLite
[0d01h35m42s] 2022-12-20T11:19:25 <DBG> [CNN] CNN Type: Digital
[0d01h35m42s] 2022-12-20T11:19:25 <DBG> [CNN] ROI #1 - TfLite
[0d01h35m42s] 2022-12-20T11:19:25 <DBG> [CNN] CNN Type: Digital
[0d01h35m43s] 2022-12-20T11:19:26 <DBG> [CNN] ROI #2 - TfLite
[0d01h35m43s] 2022-12-20T11:19:26 <DBG> [CNN] CNN Type: Digital
[0d01h35m44s] 2022-12-20T11:19:27 <DBG> [CNN] ROI #3 - TfLite
[0d01h35m44s] 2022-12-20T11:19:27 <DBG> [CNN] CNN Type: Digital
[0d01h35m44s] 2022-12-20T11:19:27 <DBG> [CNN] ROI #4 - TfLite
[0d01h35m44s] 2022-12-20T11:19:27 <DBG> [CNN] CNN Type: Digital
[0d01h35m45s] 2022-12-20T11:19:28 <DBG> [MQTT IF] Published topic: watermeter/status, content: Digitalization of ROIs (msg_id=295)
[0d01h35m45s] 2022-12-20T11:19:28 <DBG> [CNN] doFlow nach Alignment
[0d01h35m46s] 2022-12-20T11:19:29 <DBG> [TFLITE] Make Allocate
[0d01h35m46s] 2022-12-20T11:19:29 <DBG> [CNN] Processing Number 'main'
[0d01h35m46s] 2022-12-20T11:19:29 <DBG> [CNN] ROI #0 - TfLite
[0d01h35m46s] 2022-12-20T11:19:29 <DBG> [CNN] CNN Type: Analogue
[0d01h35m47s] 2022-12-20T11:19:30 <DBG> [CNN] Nach Invoke
[0d01h35m47s] 2022-12-20T11:19:30 <DBG> [CNN] ROI #1 - TfLite
[0d01h35m47s] 2022-12-20T11:19:30 <DBG> [CNN] CNN Type: Analogue
[0d01h35m47s] 2022-12-20T11:19:31 <DBG> [CNN] Nach Invoke
[0d01h35m48s] 2022-12-20T11:19:31 <DBG> [CNN] ROI #2 - TfLite
[0d01h35m48s] 2022-12-20T11:19:31 <DBG> [CNN] CNN Type: Analogue
[0d01h35m48s] 2022-12-20T11:19:31 <DBG> [CNN] Nach Invoke
[0d01h35m48s] 2022-12-20T11:19:31 <DBG> [MQTT IF] Published topic: watermeter/status, content: Post-Processing (msg_id=296)
[0d01h35m48s] 2022-12-20T11:19:31 <DBG> [CNN] getReadout _analog=0, _extendedResolution=0, prev=-1
[0d01h35m48s] 2022-12-20T11:19:31 <DBG> [CNN] ZeigerEvalAnalogNeu - kein Vorgänger - Ergebnis = 3 zahl: 3.914826 ziffer_vorgaenger = -1 AnalogFehler = 3.000000
[0d01h35m48s] 2022-12-20T11:19:31 <DBG> [CNN] ZeigerEvalAnalogNeu - Zahl uneindeutig, Korrektur nach oben - Ergebnis = 2 zahl: 1.836207 ziffer_vorgaenger = 3 AnalogFehler = 3.000000
[0d01h35m48s] 2022-12-20T11:19:32 <DBG> [CNN] ZeigerEvalAnalogNeu - Zahl uneindeutig, Korrektur nach oben - Ergebnis = 1 zahl: 1.030060 ziffer_vorgaenger = 2 AnalogFehler = 3.000000
[0d01h35m49s] 2022-12-20T11:19:32 <DBG> [CNN] getReadout _analog=0, _extendedResolution=0, prev=1
[0d01h35m49s] 2022-12-20T11:19:32 <INF> [POSTPROC] main: Raw: 02310.123, Value: , Status: Neg. Rate - Read: - Raw: 02310.123 - Pre: 2310.960
[0d01h35m49s] 2022-12-20T11:19:32 <DBG> [MQTT IF] Published topic: watermeter/status, content: Sending MQTT (msg_id=297)
[0d01h35m49s] 2022-12-20T11:19:32 <DBG> [MQTT SERVER] Publishing system MQTT topics...
[0d01h35m49s] 2022-12-20T11:19:32 <DBG> [MQTT IF] Published topic: watermeter/uptime, content: 5633 (msg_id=298)
[0d01h35m49s] 2022-12-20T11:19:32 <DBG> [MQTT IF] Published topic: watermeter/freeMem, content: 3060347 (msg_id=299)
[0d01h35m49s] 2022-12-20T11:19:32 <DBG> [MQTT IF] Published topic: watermeter/wifiRSSI, content: -84 (msg_id=300)
[0d01h35m49s] 2022-12-20T11:19:32 <DBG> [MQTT IF] Published topic: watermeter/CPUtemp, content: 55 (msg_id=301)
[0d01h35m49s] 2022-12-20T11:19:32 <DBG> [MQTT] Publishing MQTT topics...
[0d01h35m49s] 2022-12-20T11:19:32 <DBG> [MQTT IF] Published topic: watermeter/main/error, content: Neg. Rate - Read: - Raw: 02310.123 - Pre: 2310.960 (msg_id=302)
[0d01h35m49s] 2022-12-20T11:19:32 <DBG> [MQTT IF] Published topic: watermeter/main/raw, content: 02310.123 (msg_id=303)
[0d01h35m49s] 2022-12-20T11:19:32 <DBG> [MQTT IF] Published topic: watermeter/main/json, content: { "value": "", "raw": "02310.123", "pre": "2310.9600", "error".. (msg_id=304)
[0d01h35m49s] 2022-12-20T11:19:32 <DBG> [MQTT IF] Published topic: watermeter/status, content: Flow finished (msg_id=305)
[0d01h35m49s] 2022-12-20T11:19:32 <DBG> [TFLITE SERVER] CPU Temperature: 55.6
[0d01h35m49s] 2022-12-20T11:19:32 <INF> [TFLITE SERVER] Round #20 completed
[0d01h40m20s] 2022-12-20T11:24:03 <DBG> [TFLITE SERVER] ----------------------------------------------------------------
[0d01h40m20s] 2022-12-20T11:24:03 <INF> [TFLITE SERVER] Round #21 started
[0d01h40m20s] 2022-12-20T11:24:03 <DBG> [MQTT IF] Published topic: watermeter/status, content: Take Image (msg_id=306)
[0d01h40m28s] 2022-12-20T11:24:11 <DBG> [MQTT IF] Published topic: watermeter/status, content: Aligning (msg_id=307)
[0d01h40m41s] 2022-12-20T11:24:24 <DBG> [MQTT IF] Published topic: watermeter/status, content: Digitalization of ROIs (msg_id=308)
[0d01h40m41s] 2022-12-20T11:24:24 <DBG> [CNN] doFlow nach Alignment
[0d01h40m42s] 2022-12-20T11:24:25 <DBG> [TFLITE] Make Allocate
[0d01h40m42s] 2022-12-20T11:24:25 <DBG> [CNN] Processing Number 'main'
[0d01h40m42s] 2022-12-20T11:24:25 <DBG> [CNN] ROI #0 - TfLite
[0d01h40m42s] 2022-12-20T11:24:25 <DBG> [CNN] CNN Type: Digital
[0d01h40m42s] 2022-12-20T11:24:25 <DBG> [CNN] ROI #1 - TfLite
[0d01h40m42s] 2022-12-20T11:24:25 <DBG> [CNN] CNN Type: Digital
[0d01h40m43s] 2022-12-20T11:24:26 <DBG> [CNN] ROI #2 - TfLite
[0d01h40m43s] 2022-12-20T11:24:26 <DBG> [CNN] CNN Type: Digital
[0d01h40m44s] 2022-12-20T11:24:27 <DBG> [CNN] ROI #3 - TfLite
[0d01h40m44s] 2022-12-20T11:24:27 <DBG> [CNN] CNN Type: Digital
[0d01h40m44s] 2022-12-20T11:24:27 <DBG> [CNN] ROI #4 - TfLite
[0d01h40m44s] 2022-12-20T11:24:27 <DBG> [CNN] CNN Type: Digital
[0d01h40m45s] 2022-12-20T11:24:28 <DBG> [MQTT IF] Published topic: watermeter/status, content: Digitalization of ROIs (msg_id=309)
[0d01h40m45s] 2022-12-20T11:24:28 <DBG> [CNN] doFlow nach Alignment
[0d01h40m46s] 2022-12-20T11:24:29 <DBG> [TFLITE] Make Allocate
[0d01h40m46s] 2022-12-20T11:24:29 <DBG> [CNN] Processing Number 'main'
[0d01h40m46s] 2022-12-20T11:24:29 <DBG> [CNN] ROI #0 - TfLite
[0d01h40m46s] 2022-12-20T11:24:29 <DBG> [CNN] CNN Type: Analogue
[0d01h40m47s] 2022-12-20T11:24:30 <DBG> [CNN] Nach Invoke
[0d01h40m47s] 2022-12-20T11:24:30 <DBG> [CNN] ROI #1 - TfLite
[0d01h40m47s] 2022-12-20T11:24:30 <DBG> [CNN] CNN Type: Analogue
[0d01h40m48s] 2022-12-20T11:24:31 <DBG> [CNN] Nach Invoke
[0d01h40m48s] 2022-12-20T11:24:31 <DBG> [CNN] ROI #2 - TfLite
[0d01h40m48s] 2022-12-20T11:24:31 <DBG> [CNN] CNN Type: Analogue
[0d01h40m48s] 2022-12-20T11:24:31 <DBG> [CNN] Nach Invoke
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [MQTT IF] Published topic: watermeter/status, content: Post-Processing (msg_id=310)
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [CNN] getReadout _analog=0, _extendedResolution=0, prev=-1
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [CNN] ZeigerEvalAnalogNeu - kein Vorgänger - Ergebnis = 3 zahl: 3.915400 ziffer_vorgaenger = -1 AnalogFehler = 3.000000
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [CNN] ZeigerEvalAnalogNeu - Zahl uneindeutig, Korrektur nach oben - Ergebnis = 2 zahl: 1.765013 ziffer_vorgaenger = 3 AnalogFehler = 3.000000
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [CNN] ZeigerEvalAnalogNeu - Zahl uneindeutig, Korrektur nach oben - Ergebnis = 1 zahl: 1.068461 ziffer_vorgaenger = 2 AnalogFehler = 3.000000
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [CNN] getReadout _analog=0, _extendedResolution=0, prev=1
[0d01h40m49s] 2022-12-20T11:24:32 <INF> [POSTPROC] main: Raw: 0NN10.123, Value: , Status: Neg. Rate - Read: - Raw: 0NN10.123 - Pre: 2310.960
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [MQTT IF] Published topic: watermeter/status, content: Sending MQTT (msg_id=311)
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [MQTT SERVER] Publishing system MQTT topics...
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [MQTT IF] Published topic: watermeter/uptime, content: 5933 (msg_id=312)
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [MQTT IF] Published topic: watermeter/freeMem, content: 3060347 (msg_id=313)
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [MQTT IF] Published topic: watermeter/wifiRSSI, content: -85 (msg_id=314)
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [MQTT IF] Published topic: watermeter/CPUtemp, content: 55 (msg_id=315)
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [MQTT] Publishing MQTT topics...
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [MQTT IF] Published topic: watermeter/main/error, content: Neg. Rate - Read: - Raw: 0NN10.123 - Pre: 2310.960 (msg_id=316)
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [MQTT IF] Published topic: watermeter/main/raw, content: 0NN10.123 (msg_id=317)
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [MQTT IF] Published topic: watermeter/main/json, content: { "value": "", "raw": "0NN10.123", "pre": "2310.9600", "error".. (msg_id=318)
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [MQTT IF] Published topic: watermeter/status, content: Flow finished (msg_id=319)
[0d01h40m49s] 2022-12-20T11:24:32 <DBG> [TFLITE SERVER] CPU Temperature: 55.0
[0d01h40m49s] 2022-12-20T11:24:32 <INF> [TFLITE SERVER] Round #21 completed
[0d01h45m20s] 2022-12-20T11:29:03 <DBG> [TFLITE SERVER] ----------------------------------------------------------------
```

### Screenshots

the config.ini 

[MakeImage]
;LogImageLocation = /log/source
WaitBeforeTakingPicture = 5
;LogfileRetentionInDays = 15
Brightness = -2
Contrast = 0
Saturation = 0
LEDIntensity = 2
ImageQuality = 12
ImageSize = VGA
FixedExposure = false

[Alignment]
InitialRotate = 21
InitialMirror = false
SearchFieldX = 20
SearchFieldY = 20
AlignmentAlgo = default
FlipImageSize = false
/config/ref0.jpg 125 222
/config/ref1.jpg 399 131

[Digits]
Model = /config/dig-class11_1430_s2.tflite
CNNGoodThreshold = 0.5
;LogImageLocation = /log/digit
;LogfileRetentionInDays = 3
main.5-DecMigliaia 199 119 30 54 0
main.4-Migliaia 237 119 30 54 false
main.3-Cent 275 119 30 54 false
main.2-Decimal 313 119 30 54 0
main.1-Unit 351 119 30 54 0

[Analog]
Model = /config/ana-cont_11.3.0_s2.tflite
;LogImageLocation = /log/analog
;LogfileRetentionInDays = 3
main.ana1 384 219 96 96 false
main.ana2 326 304 96 96 false
main.ana3 222 326 96 96 false

[PostProcessing]
main.DecimalShift = 0
;main.AnalogDigitalTransitionStart = 
PreValueUse = true
PreValueAgeStartup = 720
;AllowNegativeRates = false
main.MaxRateValue = 1.0
;main.MaxRateType = AbsoluteChange
;main.ExtendedResolution = false
;main.IgnoreLeadingNaN = true
ErrorMessage = true
CheckDigitIncreaseConsistency = false

[MQTT]
Uri = mqtt://192.168.10.200:1883
MainTopic = watermeter
ClientID = water
user = sonoff
password = xxxxxxx
;SetRetainFlag = true
HomeassistantDiscovery = true
MeterType = water_m3

;[InfluxDB]
;Uri = undefined
;Database = 
;Measurement = undefined
;user = undefined
;password = undefined

;[GPIO]
;IO0 = input disabled 10 false false 
;IO1 = input disabled 10 false false 
;IO3 = input disabled 10 false false 
;IO4 = built-in-led disabled 10 false false 
;IO12 = input-pullup disabled 10 false false 
;IO13 = input-pullup disabled 10 false false 
LEDType = WS2812
LEDNumbers = 2
LEDColor = 150 150 150

[AutoTimer]
AutoStart = true
Intervall = 5.00

[DataLogging]
DataLogActive = true
DataLogRetentionInDays = 1

[Debug]
Logfile = 4
LogfileRetentionInDays = 1

[System]
TimeZone = CET-1CEST,M3.5.0,M10.5.0/3
TimeServer = 192.168.10.200
;AutoAdjustSummertime = false
Hostname = watermeter
SetupMode = false

**Comment by emics** [comment]
sorry, the align problem is in Round 22 and 23

`[0d01h45m20s] 2022-12-20T11:29:03 <INF> [TFLITE SERVER] Round #22 started
[0d01h45m20s] 2022-12-20T11:29:03 <DBG> [MQTT IF] Published topic: watermeter/status, content: Take Image (msg_id=320)
[0d01h45m27s] 2022-12-20T11:29:10 <DBG> [MQTT IF] Published topic: watermeter/status, content: Aligning (msg_id=321)
[0d01h45m40s] 2022-12-20T11:29:23 <DBG> [MQTT IF] Published topic: watermeter/status, content: Digitalization of ROIs (msg_id=322)
[0d01h45m40s] 2022-12-20T11:29:23 <DBG> [CNN] doFlow nach Alignment
[0d01h45m41s] 2022-12-20T11:29:24 <DBG> [TFLITE] Make Allocate
[0d01h45m41s] 2022-12-20T11:29:24 <DBG> [CNN] Processing Number 'main'
[0d01h45m41s] 2022-12-20T11:29:24 <DBG> [CNN] ROI #0 - TfLite
[0d01h45m41s] 2022-12-20T11:29:24 <DBG> [CNN] CNN Type: Digital
[0d01h45m42s] 2022-12-20T11:29:25 <DBG> [CNN] ROI #1 - TfLite
[0d01h45m42s] 2022-12-20T11:29:25 <DBG> [CNN] CNN Type: Digital
[0d01h45m43s] 2022-12-20T11:29:26 <DBG> [CNN] ROI #2 - TfLite
[0d01h45m43s] 2022-12-20T11:29:26 <DBG> [CNN] CNN Type: Digital
[0d01h45m43s] 2022-12-20T11:29:26 <DBG> [CNN] ROI #3 - TfLite
[0d01h45m43s] 2022-12-20T11:29:26 <DBG> [CNN] CNN Type: Digital
[0d01h45m44s] 2022-12-20T11:29:27 <DBG> [CNN] ROI #4 - TfLite
[0d01h45m44s] 2022-12-20T11:29:27 <DBG> [CNN] CNN Type: Digital
[0d01h45m45s] 2022-12-20T11:29:28 <DBG> [MQTT IF] Published topic: watermeter/status, content: Digitalization of ROIs (msg_id=323)
[0d01h45m45s] 2022-12-20T11:29:28 <DBG> [CNN] doFlow nach Alignment
[0d01h45m45s] 2022-12-20T11:29:28 <DBG> [TFLITE] Make Allocate
[0d01h45m45s] 2022-12-20T11:29:28 <DBG> [CNN] Processing Number 'main'
[0d01h45m45s] 2022-12-20T11:29:28 <DBG> [CNN] ROI #0 - TfLite
[0d01h45m45s] 2022-12-20T11:29:28 <DBG> [CNN] CNN Type: Analogue
[0d01h45m46s] 2022-12-20T11:29:29 <DBG> [CNN] Nach Invoke
[0d01h45m46s] 2022-12-20T11:29:29 <DBG> [CNN] ROI #1 - TfLite
[0d01h45m46s] 2022-12-20T11:29:29 <DBG> [CNN] CNN Type: Analogue
[0d01h45m47s] 2022-12-20T11:29:30 <DBG> [CNN] Nach Invoke
[0d01h45m47s] 2022-12-20T11:29:30 <DBG> [CNN] ROI #2 - TfLite
[0d01h45m47s] 2022-12-20T11:29:30 <DBG> [CNN] CNN Type: Analogue
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [CNN] Nach Invoke
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [MQTT IF] Published topic: watermeter/status, content: Post-Processing (msg_id=324)
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [CNN] getReadout _analog=0, _extendedResolution=0, prev=-1
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [CNN] ZeigerEvalAnalogNeu - kein Vorgänger - Ergebnis = 8 zahl: 8.860666 ziffer_vorgaenger = -1 AnalogFehler = 3.000000
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [CNN] ZeigerEvalAnalogNeu - Zahl eindeutig, keine Korrektur notwendig - Ergebnis = 1 zahl: 1.306791 ziffer_vorgaenger = 8 AnalogFehler = 3.000000
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [CNN] ZeigerEvalAnalogNeu - Zahl uneindeutig, Korrektur nach oben - Ergebnis = 6 zahl: 6.044638 ziffer_vorgaenger = 1 AnalogFehler = 3.000000
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [CNN] getReadout _analog=0, _extendedResolution=0, prev=6
[0d01h45m48s] 2022-12-20T11:29:31 <INF> [POSTPROC] main: Raw: NNNNN.618, Value: , Status: Neg. Rate - Read: - Raw: NNNNN.618 - Pre: 2310.960
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [MQTT IF] Published topic: watermeter/status, content: Sending MQTT (msg_id=325)
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [MQTT SERVER] Publishing system MQTT topics...
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [MQTT IF] Published topic: watermeter/uptime, content: 6232 (msg_id=326)
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [MQTT IF] Published topic: watermeter/freeMem, content: 3060347 (msg_id=327)
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [MQTT IF] Published topic: watermeter/wifiRSSI, content: -85 (msg_id=328)
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [MQTT IF] Published topic: watermeter/CPUtemp, content: 55 (msg_id=329)
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [MQTT] Publishing MQTT topics...
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [MQTT IF] Published topic: watermeter/main/error, content: Neg. Rate - Read: - Raw: NNNNN.618 - Pre: 2310.960 (msg_id=330)
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [MQTT IF] Published topic: watermeter/main/raw, content: NNNNN.618 (msg_id=331)
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [MQTT IF] Published topic: watermeter/main/json, content: { "value": "", "raw": "NNNNN.618", "pre": "2310.9600", "error".. (msg_id=332)
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [MQTT IF] Published topic: watermeter/status, content: Flow finished (msg_id=333)
[0d01h45m48s] 2022-12-20T11:29:31 <DBG> [TFLITE SERVER] CPU Temperature: 55.0
[0d01h45m48s] 2022-12-20T11:29:32 <INF> [TFLITE SERVER] Round #22 completed
[0d01h48m12s] 2022-12-20T11:31:55 <DBG> [MAIN SERVER] info_get_handler
[0d01h48m12s] 2022-12-20T11:31:55 <DBG> [MAIN SERVER] info_get_handler
[0d01h48m12s] 2022-12-20T11:31:55 <DBG> [MAIN SERVER] info_get_handler
[0d01h48m27s] 2022-12-20T11:32:10 <DBG> [MAIN SERVER] info_get_handler
[0d01h48m27s] 2022-12-20T11:32:10 <DBG> [MAIN SERVER] info_get_handler
[0d01h48m28s] 2022-12-20T11:32:11 <DBG> [MAIN SERVER] info_get_handler
[0d01h48m28s] 2022-12-20T11:32:11 <DBG> [MAIN SERVER] info_get_handler
[0d01h48m28s] 2022-12-20T11:32:11 <DBG> [MAIN SERVER] info_get_handler
[0d01h48m29s] 2022-12-20T11:32:12 <DBG> [MAIN SERVER] info_get_handler
[0d01h48m29s] 2022-12-20T11:32:12 <DBG> [MAIN SERVER] info_get_handler
[0d01h49m07s] 2022-12-20T11:32:50 <DBG> [OTA FILE] log_get_last_part_handler
[0d01h50m20s] 2022-12-20T11:34:03 <DBG> [TFLITE SERVER] ----------------------------------------------------------------
[0d01h50m20s] 2022-12-20T11:34:03 <INF> [TFLITE SERVER] Round #23 started
[0d01h50m20s] 2022-12-20T11:34:03 <DBG> [MQTT IF] Published topic: watermeter/status, content: Take Image (msg_id=334)
[0d01h50m28s] 2022-12-20T11:34:11 <DBG> [MQTT IF] Published topic: watermeter/status, content: Aligning (msg_id=335)
[0d01h50m40s] 2022-12-20T11:34:23 <DBG> [MQTT IF] Published topic: watermeter/status, content: Digitalization of ROIs (msg_id=336)
[0d01h50m40s] 2022-12-20T11:34:23 <DBG> [CNN] doFlow nach Alignment
[0d01h50m41s] 2022-12-20T11:34:24 <DBG> [TFLITE] Make Allocate
[0d01h50m41s] 2022-12-20T11:34:24 <DBG> [CNN] Processing Number 'main'
[0d01h50m41s] 2022-12-20T11:34:24 <DBG> [CNN] ROI #0 - TfLite
[0d01h50m41s] 2022-12-20T11:34:24 <DBG> [CNN] CNN Type: Digital
[0d01h50m42s] 2022-12-20T11:34:25 <DBG> [CNN] ROI #1 - TfLite
[0d01h50m42s] 2022-12-20T11:34:25 <DBG> [CNN] CNN Type: Digital
[0d01h50m43s] 2022-12-20T11:34:26 <DBG> [CNN] ROI #2 - TfLite
[0d01h50m43s] 2022-12-20T11:34:26 <DBG> [CNN] CNN Type: Digital
[0d01h50m43s] 2022-12-20T11:34:26 <DBG> [CNN] ROI #3 - TfLite
[0d01h50m43s] 2022-12-20T11:34:26 <DBG> [CNN] CNN Type: Digital
[0d01h50m44s] 2022-12-20T11:34:27 <DBG> [CNN] ROI #4 - TfLite
[0d01h50m44s] 2022-12-20T11:34:27 <DBG> [CNN] CNN Type: Digital
[0d01h50m45s] 2022-12-20T11:34:28 <DBG> [MQTT IF] Published topic: watermeter/status, content: Digitalization of ROIs (msg_id=337)
[0d01h50m45s] 2022-12-20T11:34:28 <DBG> [CNN] doFlow nach Alignment
[0d01h50m45s] 2022-12-20T11:34:28 <DBG> [TFLITE] Make Allocate
[0d01h50m45s] 2022-12-20T11:34:28 <DBG> [CNN] Processing Number 'main'
[0d01h50m45s] 2022-12-20T11:34:28 <DBG> [CNN] ROI #0 - TfLite
[0d01h50m45s] 2022-12-20T11:34:28 <DBG> [CNN] CNN Type: Analogue
[0d01h50m46s] 2022-12-20T11:34:29 <DBG> [CNN] Nach Invoke
[0d01h50m46s] 2022-12-20T11:34:29 <DBG> [CNN] ROI #1 - TfLite
[0d01h50m46s] 2022-12-20T11:34:29 <DBG> [CNN] CNN Type: Analogue
[0d01h50m47s] 2022-12-20T11:34:30 <DBG> [CNN] Nach Invoke
[0d01h50m47s] 2022-12-20T11:34:30 <DBG> [CNN] ROI #2 - TfLite
[0d01h50m47s] 2022-12-20T11:34:30 <DBG> [CNN] CNN Type: Analogue
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [CNN] Nach Invoke
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [MQTT IF] Published topic: watermeter/status, content: Post-Processing (msg_id=338)
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [CNN] getReadout _analog=0, _extendedResolution=0, prev=-1
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [CNN] ZeigerEvalAnalogNeu - kein Vorgänger - Ergebnis = 1 zahl: 1.827833 ziffer_vorgaenger = -1 AnalogFehler = 3.000000
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [CNN] ZeigerEvalAnalogNeu - Zahl uneindeutig, Korrektur nach oben - Ergebnis = 4 zahl: 4.164555 ziffer_vorgaenger = 1 AnalogFehler = 3.000000
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [CNN] ZeigerEvalAnalogNeu - Zahl eindeutig, keine Korrektur notwendig - Ergebnis = 6 zahl: 6.360424 ziffer_vorgaenger = 4 AnalogFehler = 3.000000
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [CNN] getReadout _analog=0, _extendedResolution=0, prev=6
[0d01h50m48s] 2022-12-20T11:34:31 <INF> [POSTPROC] main: Raw: N5NNN.641, Value: , Status: Rate too high - Read: 5310.641 - Pre: 2310.960 - Rate: 2999.681
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [MQTT IF] Published topic: watermeter/status, content: Sending MQTT (msg_id=339)
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [MQTT SERVER] Publishing system MQTT topics...
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [MQTT IF] Published topic: watermeter/uptime, content: 6532 (msg_id=340)
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [MQTT IF] Published topic: watermeter/freeMem, content: 3060295 (msg_id=341)
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [MQTT IF] Published topic: watermeter/wifiRSSI, content: -88 (msg_id=342)
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [MQTT IF] Published topic: watermeter/CPUtemp, content: 55 (msg_id=343)
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [MQTT] Publishing MQTT topics...
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [MQTT IF] Published topic: watermeter/main/error, content: Rate too high - Read: 5310.641 - Pre: 2310.960 - Rate: 2999.681 (msg_id=344)
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [MQTT IF] Published topic: watermeter/main/raw, content: N5NNN.641 (msg_id=345)
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [MQTT IF] Published topic: watermeter/main/json, content: { "value": "", "raw": "N5NNN.641", "pre": "2310.9600", "error".. (msg_id=346)
[0d01h50m48s] 2022-12-20T11:34:31 <DBG> [MQTT IF] Published topic: watermeter/status, content: Flow finished (msg_id=347)
[0d01h50m48s] 2022-12-20T11:34:32 <DBG> [TFLITE SERVER] CPU Temperature: 55.0
[0d01h50m49s] 2022-12-20T11:34:32 <INF> [TFLITE SERVER] Round #23 completed`

**Comment by emics** [comment]
Sorry for the wrong Issue.
thanks for your suggestion, now all is OK.

How did you do the analisys of this problem? maybe an integration in the Wiki can help other people with the same issue.

Bye
